Makefile

```diff
@@ -60,7 +60,7 @@ install:
 	[ -d $(DESTDIR)/$(SYSTEMDUTILDIR) ] || mkdir -p -m 755 $(DESTDIR)/$(SYSTEMDUTILDIR)
 
 	install -m 755 $(PROG) $(DESTDIR)/$(SBINDIR)/$(PROG)
-	ln -s ../../../$(SBINDIR)/$(PROG) $(DESTDIR)/$(SYSTEMDUTILDIR)/systemd-sysv-install
+	ln -s ../../..$(SBINDIR)/$(PROG) $(DESTDIR)/$(SYSTEMDUTILDIR)/systemd-sysv-install
```

Comment:
I know it's a bit late, but why didn't you change it to `ln -sr $(DESTDIR)/$(SBINDIR)/$(PROG) $(DESTDIR)/$(SYSTEMDUTILDIR)/systemd-sysv-install`?

Reply:
Ah, I see `-r` ends up canonicalizing the path.  Dunno.  Is that pattern used elsewhere?

Reply:
Maybe not in this package, but I've seen it in other ones. It's a much better pattern than the one this thing uses now...